Sources/DittoPresenceDegradation/NewSessionView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SwiftUIView.swift
+//  
+//
+//  Created by Walker Erekson on 2/13/24.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+public struct NewSessionView: View {
+    
+    @Binding private var expectedPeers: Int
+    @Binding private var apiEnabled: Bool
+    @Binding var isPresented: Bool
+    @Binding var sessionStartTime: String?
+    var onDismiss: () -> Void
+    
+    public init(expectedPeers: Binding<Int>, apiEnabled: Binding<Bool>, isPresented: Binding<Bool>, sessionStartTime: Binding<String?>, onDismiss: @escaping () -> Void) {
+        self._expectedPeers = expectedPeers
+        self._apiEnabled = apiEnabled
+        self._isPresented = isPresented
+        self._sessionStartTime = sessionStartTime
+        self.onDismiss = onDismiss
+    }
+    
+    public var body: some View {
+        VStack(alignment: .leading) {
+            HStack() {
+                Text("Expected peer count in the mesh:")
+                TextField("0", value: $expectedPeers, formatter: NumberFormatter())
+                    .keyboardType(.numberPad)
+                    .frame(maxWidth: 50)
+                    .onChange(of: expectedPeers) { newValue in
+                        if newValue < 0 {
+                            expectedPeers = 0 
+                        }
```

Comment:
Does it make sense to have expectedPeers be a `UInt` to avoid this dance? That might be a cleaner way to prevent negative peer counts if we're trying to enforce that.

Reply:
That still wouldn't stop users from being able to input negative or non numbers into the textfield. For a cleaner UI I have added an alert that will notify the user if they try to save the session with anything but a number greater than 0